types/qunit/index.d.ts

```diff
@@ -637,7 +637,7 @@ declare global {
          * @param {string} Title of unit being tested
          * @param callback Function to close over assertions
          */
-        test(name: string, callback: (assert: Assert) => void | Promise<any>): void;
+        test(name: string, callback: (assert: Assert) => void | Promise<void>): void;
```

Comment:
It would be nice to add some tests for these (that I realize don't actually test anything, yet, but one day...)

Reply:
I had tests, but I think you blew them away in a rebase :P

Reply:
Ah, they are still over in ember-qunit:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/01079a9b7d56ef012bfaa2eac24f3aaffd023571/types/ember-qunit/ember-qunit-tests.ts#L118-L164

Reply:
Ah, I thought I had some here too, but maybe not.

Reply:
Sorry if I *did* blow them away. I don't think my PR touched tests, though.